Developers: Contribution Guidelines (#3801)
joernio/joern: README.md

* Try make use of the templates as far as possible, however they may not suite all needs. The minimum we would like to see is:
    - A title that briefly describes the change and purpose of the PR, preferably with the affected module in square brackets, e.g. `[javasrc2cpg] Addition Operator Fix`.
    - A short description of the changes in the body of the PR. This could be in bullet points or paragraphs.
    - A link or reference to the related issue, if any exists.
* Do not:
    - Immediately CC/@/email spam other contributors, the team will review the PR and assign the most appropriate contributor to review the PR. Joern is maintained by industry partners and researchers alike, for the most part with their own goals and priorities, and additional help is largely volunteer work. If your PR is going stale, then reach out to us in follow-up comments with @'s asking for an explanation of priority or planning of when it may be addressed (if ever, depending on quality).
    - Leave the description body empty, this makes reviewing the purpose of the PR difficult.

> * Remember to format your code with `scalafmt`. Use `sbt scalafmt test:scalafmt`

Reviewer: ```suggestion
* Remember to format your code, i.e. run `sbt scalafmt Test/scalafmt`
```
